mvlearn/preprocessing/repeat.py

```diff
@@ -0,0 +1,201 @@
+"""Filtering module."""
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Pierre Ablin
+
+from sklearn.base import clone, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+from ..utils import check_Xs
+
+
+class RepeatTransform(TransformerMixin):
+    r"""Apply a sklearn transformer to each view of a dataset
+
+    Build a transformer from multiview dataset to multiview dataset by
+    using individual scikit-learn transformer on each view.
+
+    Parameters
+    ----------
+    transformer : a sklearn transformer instance, or a list
+        Either a single sklearn transformer that will be applied to each
+        view. One clone of the estimator will correspond to each view.
+        Otherwise, it should be a list of estimators, of length the number of
+        views in the multiview dataset.
+
+    Attributes
+    ----------
+    n_views_ : int
+        The number of views in the input dataset
+
+    transformer_list_ : list of objects of length n_views_
```

Comment:
```suggestion
    transformers_ : list of objects of length n_views_
```

I would prefer this. it's more consistent with sklearn that uses estimators_